src/components/Menu/TopCodeMenu.tsx

```diff
@@ -7,13 +7,18 @@ import VersionMenu, { VersionMenuProps } from './VersionMenu';
 
 const TopCodeMenu = ({ languages, versionData }: { languages: string[]; versionData: VersionMenuProps }) => {
   const pageLanguage = useContext(PageLanguageContext);
-  const showCodeMenu = languages && languages.length > 1;
+  const showVersionMenu = versionData && versionData.versions.length > 0;
+  const showLanguageSelector = languages && languages.length > 0;
```

Comment:
Are the guards needed here?
Can it just be
`languages.length > 0` because of the TS guards?